src/helpers/daily-data-providers/combined-sleep-v2.ts

```diff
@@ -18,7 +18,7 @@ export default async function (startDate: Date, endDate: Date): Promise<DailyDat
     if (settings.garminEnabled) {
         providers.push(garminTotalSleepMinutesDataProvider(startDate, endDate));
     }
-    if (settings.appleHealthEnabled && settings.queryableDeviceDataTypes.some(ddt => ddt.namespace === "AppleHealth" && ddt.type === "SleepAnalysisInterval")) {
+    if (settings.appleHealthEnabled && deviceDataV2Types.some(ddt => ddt.namespace === "AppleHealth" && ddt.type === "Sleep Analysis")) {
```

Comment:
Can we just drop this V2 sleep thing and use V1 sleep for glucose chart?

Reply:
Yes, that should be fine.

Do you think we should update the single unified sleep provider to favor V2 querying when available?  At present, it uses V1 for Fitbit, Garmin, and Apple Health sleep.  We have V2 data providers available for those, so we could check the available V2 data types to determine which one to use for each source.

Reply:
I have dropped the combined-sleep-v2 data provider and updated the glucose chart to use the normal unified sleep provider.

I'm going to go ahead and merge this now, but will open a follow up PR if we'd like to update the combined sleep provider to support V2 data querying for Fitbit, Garmin, and AH sleep.